.github/workflows/gh-to-jira.yaml

```diff
@@ -0,0 +1,77 @@
+name: Create Jira Issue from GH Release
+run-name: GH Release to Jira.
+on:
+  workflow_dispatch:
+    inputs:
+      component:
+        default: 'Data Science Pipelines'
+        description: 'ODH Component'
+        required: true
+      target_release:
+        description: 'Target Downstream Release'
+        required: true
+      gh_org:
+        default: 'opendatahub-io'
+        description: 'Upstream GH Org'
+        required: true
+      repos:
+        default: |
+          [{"repo_name":"data-science-pipelines","target_release":"UPDATE","previous_release":"UPDATE"},{"repo_name":"data-science-pipelines-operator","target_release":"UPDATE","previous_release":"UPDATE"}]
+        description: 'Upstream Source Repos & Tags'
+        required: true
+      labels:
+        default: 'qe/verify'
+        required: true
+        description: ""
+      jira_server:
+        default: 'https://issues.redhat.com'
+        required: true
+        description: "Jira Server"
+      jira_project:
+        default: "RHODS"
+        required: true
+        description: "Jira Project"
+      jira_labels:
+        default: "MLOps"
+        required: true
+        description: "Jira Labels to Add"
+      jira_issue_type:
+        default: "Story"
+        required: true
+        description: "Jira Issue Type"
+      jira_priority:
+        default: 'Normal'
+        required: true
+        description: "Jira Priority to Set"
+
+jobs:
+  gh-to-jira:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          repository: HumairAK/gh-to-jira
+          fetch-depth: '0'
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.10'
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+      - name: Submit Jira
+        env:
+          GITHUB_TOKEN: ${{ secrets.GTJ_GH_TOKEN }}
+          JIRA_TOKEN: ${{ secrets.GTJ_JIRA_TOKEN }}
```

Comment:
how do these work? Whose tokens are these and where are they stored?

Reply:
GH token is a fine grained token from ODH, jira token is mine for now, we'll need to look into bot tokens to replace with, which can be a next step.